python-checks/src/main/java/org/sonar/python/checks/regex/VerboseRegexCheck.java

```diff
@@ -66,4 +70,25 @@ public PreciseIssue addIssue(RegexSyntaxElement regexTree, String message, @Null
       })
       .orElse(null);
   }
+
+  private class PythonVerboseRegexRangeCheckVisitor extends RegexBaseVisitor {
```

Comment:
Do you think it would make sense to have this part in `analyzer-commons` instead?
I can see this issue being relevant for other languages than Python, and I think the implementation wouldn't have to change, right?
To clarify, this is a genuine question, as I'm not sure we want to make this change in other analyzer's behaviors without their respective bubbles knowing about it.